doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -52,14 +52,17 @@ Ensure date.timezone is set in php.ini to your preferred time zone.  See http://
     vim /etc/php/7.0/fpm/php.ini
     vim /etc/php/7.0/cli/php.ini
 
+    vi /etc/php/7.0/fpm/pool.d/www.conf
+    Change the user and group from www-data to librenms
```

Comment:
Would it be best to make a pool just for librenms?

Reply:
Yeah, if anything it would be better to create separate pool just for LibreNMS just so that default one doesn't run as librenms when user installs something else on same server.

Reply:
I suspect that people who run this on a shared server won't be needing the install manual. Other config is an option, but what about the default config? Do you remove it? Do you leave it and have people run a php pool which doesn't do anything? So using "resources"?

Reply:
Install manual is always relevant - you don't expect people to just magically know know that they need to point `/api/v0` to `/api_v0.php`, don't you? Further, I would raise argument that less knowledgeable people would be more willing to install it on shared server, and people who 'know stuff' setting it on separate VM/LXC container. I don't see leaving default pool as an issue really.

Reply:
One long term item is we need to have PHP create files as the librenms user.  Because there will be files shared between the frontend and backend.  The backend always runs as the librenms user.

Reply:
So, do i change the docs to create a new pool ?

Reply:
Sure